Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -11937,8 +11943,15 @@ def pan_os_get_running_config(args: dict):
     if args.get("target"):
         params["target"] = args.get("target")
 
+    file_name_arg = args.get("filename")
+    target = args.get("target")
+    if file_name_arg != 'running_config':
+        file_name =  target + '_' + file_name_arg + '_running_config'
```

Comment:
Let's use the filename as we receive it from the user without changing it.

Reply:
Hi @jlevypaloalto We will have to append at least the target name to the filename argument because it will save all the config files to all the hostnames with the same filename which will be confusing.